bin/build-ember

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+set -e
+
+rm -rf build/ember-app
+cd src/ember-app
+./node_modules/ember-cli/bin/ember build --output-path ../../build/ember-app/
```

Comment:
This build line should probably include the environment flag to build a production version of Ember:

    --environment=production

Without it, the assets built are quite large. It may need to further complexities on integrating with Ember though ...